README.md

```diff
@@ -6,6 +6,7 @@
 [![Aqua QA](https://raw.githubusercontent.com/JuliaTesting/Aqua.jl/master/badge.svg)](https://github.com/JuliaTesting/Aqua.jl)
 [![Coverage](https://codecov.io/gh/albertomercurio/QuantumToolbox.jl/branch/main/graph/badge.svg)](https://codecov.io/gh/albertomercurio/QuantumToolbox.jl)
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.11096277.svg)](https://doi.org/10.5281/zenodo.11096277)
+![Fancy logo](./docs/src/assets/logo.png)
```

Comment:
Since we doesn't have individual logos for light- and dark-themes Maybe just use `![Logo](./docs/src/assets/logo.png)`